plugins/pencilblue/controllers/actions/salesforce/login.js

```diff
@@ -13,7 +13,12 @@ module.exports = function LoginSFSSOControllerModule(pb) {
     class LoginSFSSOController extends pb.BaseController {
         render(cb) {
             this.sanitizeObject(this.body);
-            this.salesforceSSO(cb);
+            if(this.req.session && this.req.session.authentication && this.req.session.authentication.user &&
+                this.req.session.authentication.user.external_user_id){
+                this.redirect (`https://${this.req.headers.host}/profile/view`,cb);
```

Comment:
What is going to happen in the context of a reverse proxy? Will this.req.headers.host contain the proper value?